mappings/net/minecraft/item/Items.mapping

```diff
@@ -88,6 +88,7 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_20392 MOSSY_COBBLESTONE Lnet/minecraft/class_1792;
 	FIELD field_20393 PURPUR_BLOCK Lnet/minecraft/class_1792;
 	FIELD field_20394 GRANITE Lnet/minecraft/class_1792;
+	FIELD field_20395 STONE_BRICKS Lnet/minecraft/class_1792;
```

Comment:
still not sure why this mapping was dropped even after the intermediary fix but eh

Reply:
It should now be fixed, I had to do a hotfix to intermediary, please make sure this has the old intermediary name and not a new one.

Reply:
This is after the intermediary fix, this is the old intermediary name. The mapping was dropped for some reason.